.github/workflows/dependabot-build.yaml

```diff
@@ -6,10 +6,8 @@ on:
 
 jobs:
   build:
+    runs-on: ubuntu-latest
     if: github.actor == 'dependabot[bot]'
-    strategy:
-      matrix:
-        environment: [iota, shimmer, next]
-    uses: ./.github/workflows/build.reusable.yaml
-    with:
-      environment: ${{ matrix.environment }}
+    steps:
+      - name: build
+        uses: ./.github/actions/build/
```

Comment:
Do we need a checkout step before this? How else will it find the composite action?

Reply:
True that. Thanks

Reply:
Fixed in b2f6dc5